DEVELOPMENT.md

```diff
@@ -1,77 +1,138 @@
-- [Local builds](#local-builds)
-- [Command documentation](#command-documentation)
-- [Unit testing](#unit-testing)
-- [Acceptance testing](#acceptance-testing)
-  - [Installing Cartographer](#installing-cartographer)
-- [Debug Code Completion](#debug-code-completion)
+# Development
 
----
+This doc explains how to set up a development environment so you can get started
+[contributing](CONTRIBUTING.md) to `Tanzu apps plugin`. Also
+take a look at:
 
-## Local builds
+- [How to add and run tests](#testing)
+- [Iterating](#iterating)
 
-Building the CLI plugin locally requires both go 1.18 and the [Tanzu CLI](https://github.com/vmware-tanzu/tanzu-framework/tree/main/cmd/cli#installation) with the builder plugin installed.
+## Prerequisites
 
-Install the Tanzu builder and test plugins:
+Follow the instructions below to set up your development environment. Once you
+meet these requirements, you can make changes and
+[deploy your own version of apps plugin](#starting-apps-plugin)!
 
-```sh
-tanzu init
-```
+Before submitting a PR, see also [CONTRIBUTING.md](./CONTRIBUTING.md).
 
-```sh
-tanzu plugin repo add -b tanzu-cli-admin-plugins -n admin -p artifacts-admin
-```
+### Install requirements
 
-```sh
-tanzu plugin install builder
+You must install these tools:
+
+- [`go`]: for compiling the plugin as well as other dependencies - 1.16+
```

Comment:
**suggestion**: Should we tell go 1.18?

Reply:
I thought about that too. We are using 1.18 but technically they can build from 1.16. I am open to 1.18 suggestion as well

Reply:
I don't have a strong preference either way. Just want to keep it same as go.mod